### DISCUSSION TITLE
Customizable automatic welcome message to the session.

### DISCUSSION THREAD

**Discussion by SeeeSe** [YimMenu/YimMenu#3551] [Ideas]
Be able to set a welcome message to the session for things like: example "This session is protected by User_A, grieffers will be kicked", "Commands are enabled, type /spawn and spam your vehicle".
If I'm not mistaken, there is an option to send a direct message via chat, use that to send the welcome messages.  (Google translator)

**Comment by V1RU513N** [comment]
It's useless, and probably won't be added to the menu.
but it can be done with a lua script.

**Reply by SeeeSe** [reply]
I don't see it as useless. We already have reactions that are reported via chat, I don't see anything wrong with it and I don't think it would be that difficult to add.
I have no idea how lua scripts work.
Apart from that it can add a lot of game depending on how it is used.

**Comment by iwanttodo1t** [comment]
If you can't detect all cheaters, don't do it.

**Reply by iwanttodo1t** [reply]
There is no menu to detect all cheaters, and cheaters can attack everyone else without you knowing.

**Reply by SeeeSe** [reply]
> There is no menu to detect all cheaters, and cheaters can attack everyone else without you knowing.

What does one thing have to do with the other? Besides, it helps to kick out toxic modders, when they see that they are exposed they proceed to attack you, so you can kick them out.

**Reply by iwanttodo1t** [reply]
keep fantasizing.

**Reply by SeeeSe** [reply]
You should lower your ego, you're going to hurt yourself. Greetings, friend.
